ister.py

```diff
@@ -768,7 +790,12 @@ def get_template(template_location):
     """Fetch JSON template file for installer
     """
     json_file = request.urlopen(template_location)
-    return json.loads(json_file.read().decode("utf-8"))
+    parsed_json = json.loads(json_file.read().decode("utf-8"))
+    try:
```

Comment:
I think better to just do ->
if not parsed_json.get("PackageManager"):
    parsed_json["PackageManager"] = SWUPD

Reply:
And I'm curious why this is done here instead of in validate_template?

Reply:
I was thinking of defaulting the PackageManager to swupd as close to the source as possible so that validate_template would be strictly checking if the template that was supplied was all right instead of possibly modifying it with a default value.

Reply:
Ah, so that's what the get function is used for.  I was initially doing `if not parsed_json["PackageManager"]` but getting a KeyError exception.  I Googled badly for that one.

Reply:
Okay, that's fine with me then.